pkg/probe/bgp_neighbors_ipv4.go

```diff
@@ -0,0 +1,66 @@
+package probe
+
+import (
+	"log"
+	"strconv"
+    
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/bluecmd/fortigate_exporter/internal/version"
+)
+
+func probeBgpNeighborsIPv4(c http.FortiHTTP) ([]prometheus.Metric, bool) {
+
+	var (
+		bpgNeighbor = prometheus.NewDesc(
+			"fortigate_bgp_neighbors_ipv4",
+			"Confiured bgp neighbors over ipv4",
+			[]string{"vdom", "remote_as", "state", "admin_status", "local_ip", "neighbor_ip", "type"}, nil,
```

Comment:
Type is redundant, it is only `ipv4` or `ipv6` which is implied by the `_ipv4` metric name.